edgelet/iotedge/src/check/checks/well_formed_config.rs

```diff
@@ -12,10 +12,10 @@ pub(crate) struct WellFormedConfig {}
 
 impl Checker for WellFormedConfig {
     fn id(&self) -> &'static str {
-        "config-yaml-well-formed"
+        "config-toml-well-formed"
```

Comment:
It's not a single TOML here either. Just make it `config` and `"configuration"` like in your i-i-s PR.

Reply:
Yep, sorry, that's my pre-lunch hunger manifesting itself.